### PR TITLE
Updated FAQ

### PR DESCRIPTION
**Pull request by dedepete**
Discussions are welcome.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -571,39 +571,37 @@
...
         <![CDATA[
-        <h5>• Why can\'t I see my <b>Organizations</b> either <i>Private</i> or <i>Public</i> ones?</h5>
-        <p>Open up https://github.com/settings/applications and look for FastHub, open it then scroll to Organization access and click on Grant Button,
-        alternatively login via <b>Access Token</b> which will ease this setup.</p>
+        <h5>• Why can I see neither my <i>Private</i> nor my <i>Public</i> <b>Organizations</b>?</h5>
+        <p>Open up https://github.com/settings/applications and look for FastHub, open it up, scroll down to "Organization access" and click on the "Grant" button.
+        Alternatively you can sign in using <b>Access Token</b> which will ease the process.</p>
 
-        <h5>• I tried to login via Access Token &amp; OTP but it does not work?</h5>
-        <p>You can\'t login via Access Token &amp; OTP all together due to the lifetime of the OTP code, you\'ll be required to login in every few seconds.</p>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: with `an` access

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -571,39 +571,37 @@
...
+        <h5>• I have tried signing in with Access Token and One-Time Password (OTP) without any luck. What's wrong?</h5>
+        <p>You can\'t sign in using Access Token and OTP all together. Due to the limited lifetime of such passwords you\'ll have to repeat a sign in procedure every few seconds.</p>
 
-        <h5>• Why are my Private Repo &amp; Enterprise Wiki not showing up?</h5>
-        <p>It\'s due to FastHub scraping GitHub Wiki page &amp; Private Repos require session token that FastHub is unable to obtain.</p>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: aren't not isn't

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -571,39 +571,37 @@
...
+        <h5>• Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?</h5>
+        <p>As FastHub scraps GitHub's Wiki pages, the application is unable to obtain any session token which is required by Private Repositories.</p>
 
-        <h5>• I login with Enterprise account but can\'t interact with anything other than my Enterprise GitHub?</h5>
-        <p>Well, logically, you can\'t access anything else other than your Enterprise.  FastHub tries to allow as much possible, but can\'t do much about it
-        in most cases, since your login credential doesn\'t exists in GitHub server. But in a <b>few</b>
-        cases your GitHub account Oauth token will do the trick.</p>
+        <h5>• I have signed into my Enterprise account. Why can\'t I interact with anything other than my Enterprise GitHub?</h5>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: its either "do anything" or "do much" not both

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -571,39 +571,37 @@
...
+        <p>Well, logically, you can\'t access anything else other than your Enterprise. FastHub tries to require as much permissions as possible, but can\'t do anything much about it
+        in most cases, since your credentials are not on GitHub server. But in <b>some</b> cases your GitHub account OAuth token will do the trick.</p>
 
-        <h5>• Why am I having problems editing Issues/PRs?</h5>
-        <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
+        <h5>• Why am I having problems editing Issues/Pull Requests (PRs)?</h5>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: take out the word "then"

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Agreed. `A` is also required for OTP.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `Isn't` is correct here, because we're meaning only one Wiki here.  
`Repository's Wiki`

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: I ment `more`. Will correct.

**Comment by dedepete** [comment]
@yakov116 btw Travis has started to fail after bfa9e7b. Any ideas what is wrong?

**Comment by yakov116** [comment]
@dedepete github is having issues

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Nope you are saying private OR enterprise = 2

**Comment by dedepete** [comment]
@yakov116 so everything is fine now?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -571,39 +571,37 @@
     <string name="please_read">PLEASE READ!</string>
     <string name="fasthub_faq_description">
         <![CDATA[
-        <h5>• Why can\'t I see my <b>Organizations</b> either <i>Private</i> or <i>Public</i> ones?</h5>
-        <p>Open up https://github.com/settings/applications and look for FastHub, open it then scroll to Organization access and click on Grant Button,
-        alternatively login via <b>Access Token</b> which will ease this setup.</p>
```
Comment (k0shk0sh) [inline app/src/main/res/values/strings.xml]: why can't I instead of why I can see.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -571,39 +571,37 @@
...
-        <h5>• Why am I having problems editing Issues/PRs?</h5>
-        <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
+        <h5>• Why am I having problems editing Issues/Pull Requests (PRs)?</h5>
+        <p>If you are unable to edit Issues/PRs in a Public Organization's repository, please contact your Organization to grant access to FastHub. Alternatively you can sign in with an Access Token.</p>
 
-        <h5>• I\'m having this issue or I want this &amp; that!!</h5>
-        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create new issue for bugs or feature requests, I really do encourage you to
-        search before opening a ticket. Any duplicate request will result in it being closed immediately.</p>
+        <h5>• I\'m having problems! / I want to suggest you some things!</h5>
+        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create a new issue for any bugs and feature requests. I really do encourage you to
```
Comment (k0shk0sh) [inline app/src/main/res/values/strings.xml]: labeled as duplicated and not invalid.

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `not` is a part of `neither`.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Resolved in bc6ef01.

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Don't like the nether, too fancy.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: `be required` i think is better

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -571,39 +571,37 @@
...
+        Alternatively you can sign in using <b>Access Token</b> which will ease the process.</p>
 
-        <h5>• I tried to login via Access Token &amp; OTP but it does not work?</h5>
-        <p>You can\'t login via Access Token &amp; OTP all together due to the lifetime of the OTP code, you\'ll be required to login in every few seconds.</p>
+        <h5>• I have tried signing in with an Access Token and a One-Time Password (OTP) without any luck. What's wrong?</h5>
+        <p>You can\'t sign in using Access Token and OTP all together. Due to the limited lifetime of such passwords you\'ll have to repeat a sign in procedure every few seconds.</p>
 
-        <h5>• Why are my Private Repo &amp; Enterprise Wiki not showing up?</h5>
-        <p>It\'s due to FastHub scraping GitHub Wiki page &amp; Private Repos require session token that FastHub is unable to obtain.</p>
+        <h5>• Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?</h5>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: I still think `aren't` is the correct way to say it

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -571,39 +571,37 @@
...
+        <p>As FastHub scraps GitHub's Wiki pages, the application is unable to obtain any session token which is required by Private Repositories.</p>
 
-        <h5>• I login with Enterprise account but can\'t interact with anything other than my Enterprise GitHub?</h5>
-        <p>Well, logically, you can\'t access anything else other than your Enterprise.  FastHub tries to allow as much possible, but can\'t do much about it
-        in most cases, since your login credential doesn\'t exists in GitHub server. But in a <b>few</b>
-        cases your GitHub account Oauth token will do the trick.</p>
+        <h5>• I have signed into my Enterprise account. Why can\'t I interact with anything other than my Enterprise GitHub?</h5>
+        <p>Well, logically, you can\'t access anything else other than your Enterprise. FastHub tries to require as much permissions as possible, but can\'t do anything more about it
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: as `many` permissions

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -571,39 +571,37 @@
...
+        in most cases, since your credentials are not on GitHub server. But in <b>some</b> cases your GitHub account OAuth token will do the trick.</p>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: github`'s` server

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: Take out the word `do` before the word `encourage`

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -571,39 +571,37 @@
...
-        <li>Rate FastHub in the Play Store</li>
+        <li>Your student card and your identity card/passport (that shows your name and face to compare them!);</li>
+        <li>Your education start and graduation dates.</li>
         </ul>
-        <p>If you aren\'t a student and you can\'t afford to pay for PRO, you\'ll need:</p>
+        <p>If you aren\'t a student and you can\'t afford purchasing PRO, you\'ll need to:</p>
         <ul>
-        <li>Write an article about FastHub in social media such as (Medium)</li>
-        <li>Rate FastHub in the Play Store</li>
+        <li>Write an article about FastHub in any social media such as (Medium);</li>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: the `(` should start before such as

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -571,39 +571,37 @@
...
-        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create new issue for bugs or feature requests, I really do encourage you to
-        search before opening a ticket. Any duplicate request will result in it being closed immediately.</p>
+        <h5>• I\'m having problems! / I want to suggest you some things!</h5>
+        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create a new issue for any bugs and feature requests. I really do encourage you to
+        search for similar issues before opening a new one. Any duplicated ticket will be labeled as duplicated and closed immediately.</p>
 
-        <h5>• How do I get PROMO CODE?</h5>
-        <p>If you are a student, you\'ll have to provide me via Email that you are student, you will need below documents:</p>
+        <h5>• How can I get a PROMO CODE?</h5>
+        <p>If you are a student, you\'ll have to prove me that you are student. You will need to send me via Email some proofs listed below:</p>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: `you are student.` take that out.

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Don't agree. This line could be extended with additional social media. 
E.g. `Write an article about FastHub in any social media such as (Medium), (Twitter), (Facebook), (VK), etc.`

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: There's only a single Wiki in the repo. Usage of `aren't` is contextually incorrect.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `do` adds an emphasis to the statement.

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: (such as Medium, Twitter, Facebook, VK), 😄

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: doesn't mean its correct english

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: But the links are different for the websites.

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: so what? such as belong inside the parentheses

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Nope disagree. You are talking about **both** Private AND Enterprise.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: But it is so.

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Not that i know of

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: A privare repo OR an enterprise repo.  
Here's how the sentences look like without OR:
`Why isn\'t my Private Repository\'s Wiki showing up?` AND `Why isn\'t my Enterprise Repository\'s Wiki showing up?`

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: That would be the correct way to use isn't. since you are putting both in one is uses `aren't`

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `[Medium](url), [Twitter](url), [Facebook](url), [VK](url), etc.`

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: the words ok understand change it to links....

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Okey, another example.  
Let's remove the adjectives which describe a repository's state.  
`Why isn\'t my Repository\'s Wiki showing up?`  
See? There's only a single repo, not two.

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: that would be correct

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: I don't get the meaning of parentheses here. This is how I understand them.

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Let's add them back again.  
`Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?`  
A single repo remains.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: Here's a good explanation:
https://www.italki.com/question/319207?hl=en

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: OK you push your update and i will fix if i have an issue

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Yes that makes sense if your responding to someone. (look at the example) but normally its not used..

Whatever...

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: No :) now you are saying 2 things LOL....
isn't would be correct if it would be written like this 

Why isn\'t my Private or my Enterprise

Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `/` reduces `or my` in this sentence.

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: / = `or` not `or my` LOL what ever....

Comment (dedepete) [reply app/src/main/res/values/strings.xml]: I'm going to edit this sentence into
`Why aren\'t my Private and Enterprise Repositories\' Wikis showing up?`

Comment (dedepete) [reply app/src/main/res/values/strings.xml]: I was wrong. `/` reduces `or`, not `or my`. But still we were talking about one repo's wiki, so my variant was absolutely correct.

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -593,7 +593,7 @@
         search for similar issues before opening a new one. Any duplicated ticket will be labeled as duplicated and closed immediately.</p>
 
         <h5>• How can I get a PROMO CODE?</h5>
-        <p>If you are a student, you\'ll have to prove me that you are student. You will need to send me via Email some proofs listed below:</p>
+        <p>If you are a student, you\'ll have to prove me that. You will need to send me via Email some proofs listed below:</p>
```
Comment (yakov116) [inline app/src/main/res/values/strings.xml]: Prove that to me

**Comment by yakov116** [comment]
@dedepete comment on a line i cant see where your commenting on

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -593,7 +593,7 @@
         search for similar issues before opening a new one. Any duplicated ticket will be labeled as duplicated and closed immediately.</p>
 
         <h5>• How can I get a PROMO CODE?</h5>
-        <p>If you are a student, you\'ll have to prove that to me. Proof can be sent via Email as listed below:</p>
+        <p>If you are a student, you\'ll have to prove it to me. Proof could be be sent via Email as listed below:</p>
```
Comment (dedepete) [inline app/src/main/res/values/strings.xml]: `Proofs`.

**Comment by dedepete** [comment]
@yakov116 it was about parentheses.

File: app/src/main/res/values/strings.xml (hunk 13 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Nope. We are using it as a general term.

**Comment by k0shk0sh** [comment]
@yakov116 do not commit directly to a PR, only when its merged, otherwise do not commit directly to other prs without them approving it......

File: app/src/main/res/values/strings.xml (hunk 13 above)
Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `A passport` is a `proof`. `A passport with a student card` are `proofs`.

Comment (yakov116) [reply app/src/main/res/values/strings.xml]: A passport is one type of proof. 
Google `proofs definition`

I don't even think `proofs` is a word...

Comment (dedepete) [reply app/src/main/res/values/strings.xml]: `Proofs` is a plural `proof`.

**Comment by dedepete** [comment]
Alright, I have pushed some changes again. Let @k0shk0sh decide if `(Medium)` is correct or not.

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Let me explain. 

Parentheses are user when you want to add something that is not in context. The sentence should make sense without it. The use is to explain what mean with something you said.

So in our case if you take out the parentheses (including its word), the sentence will not make sense.

`Write an article about FastHub in any social media such as`

> So in our case if you take out the parentheses (including its word), the sentence will not make sense.
In the above case i used parentheses lets take it out and my sentence will make sense.
>So in our case if you take out the parentheses, the sentence will not make sense.

File: app/src/main/res/values/strings.xml (hunk 13 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: Please link me to where that word exists, Proof is a noun.

http://www.dictionary.com/browse/proofs

**Comment by k0shk0sh** [comment]
Wait unit I approve this and if it's approved then we don't need to change anything

File: app/src/main/res/values/strings.xml (hunk 13 above)
Comment (yakov116) [reply app/src/main/res/values/strings.xml]: @dedepete ?

**Comment by yakov116** [comment]
@k0shk0sh if you approve please squash and merge

File: .travis.yml [hunk 14]
```diff
@@ -20,6 +20,7 @@ before_install:
 
 before_script:
   - echo $ANDROID_HOME
+  - (while sleep 3; do echo "y"; done) | $ANDROID_HOME/tools/bin/sdkmanager "build-tools;26.0.2" "platforms;android-26"
```
Comment (k0shk0sh) [inline .travis.yml]: Can you separate this into another PR?

Comment (dedepete) [reply .travis.yml]: Sure, but a bit later.

Comment (k0shk0sh) [reply .travis.yml]: Sure, take your time :) I'll look at this later after work :)

Comment (dedepete) [reply .travis.yml]: @k0shk0sh done. #1134

**Comment by yakov116** [comment]
@dedepete revert the travis commit.

**Comment by dedepete** [comment]
@yakov116 Well, I have just made a commit, which reverts that fix. ¯\_(ツ)_/¯  
But I can hard reset it, if needed.

**Comment by yakov116** [comment]
@dedepete No need since we will squash and merge
